cumulusci/core/config/SfdxOrgConfig.py

```diff
@@ -137,7 +137,6 @@ def force_refresh_oauth_token(self):
 
```

Comment:
While you are in this code can you look at the function above? A) The comment on Line 125 doesn't seem right in several ways,
B) the stdout_list should only be parsed in the "if" block and C) I'm just taking it "on faith" that this function which doesn't write to self.anything at all is nevertheless doing everything it needs to do...just forcing a refresh at the SFDX level. Is it properly named?

Reply:
@prescod

A and B -- I agree, but not in the scope of this PR
C - Yes, it just forces a refresh at the sfdx level. It is only called from `SfdxOrgConfig.refresh_oauth_token`, which first deletes `self._sfdx_info` so that future attempts to get the access token will re-fetch info from sfdx. It wouldn't hurt to move clearing _sfdx_info into `force_refresh_oauth_token` so that it can't be missed.